src/util.ts

```diff
@@ -0,0 +1,87 @@
+import { Logger } from './logger';
+
+const temp = require('temp');
+const fs = require('fs');
+const logger:Logger = Logger.getInstance();
+
+
+export class TempFileHandler {
+    track: any;
+
+    create(TextEditor) {
+        let tempFile = temp.openSync({suffix: '.py'});
```

Comment:
I read it one more time. The issue is not valid because temp directory is created by me and doesn't contain - char.